torch/onnx/symbolic_opset9.py

```diff
@@ -2039,3 +2039,21 @@ def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
     # Norm has shape [N, C, *] so we reshape weight and bias to [C, *]
     axes = [i for i in range(1, len(input_sizes) - 1)]
     return add(g, mul(g, norm, g.op("Unsqueeze", weight, axes_i=axes)), g.op("Unsqueeze", bias, axes_i=axes))
+
+@parse_args('v', 'f', 'i', 't')
+def quantize_per_tensor(g, input, scale, zero_point, dtype):
+    kwargs = {
+        "Y_scale_f": scale,
+        "zero_point_i": zero_point,
+    }
+    output = g.op("_caffe2::Int8Quantize", input, **kwargs)
```

Comment:
These two functions are caffe2 specific, let's move it to symbolic_caffe2.py first. Later we can add the official support for ONNX.

Reply:
These ops are in the `aten::` namespace (not `quantized::` namespace) so they are registered using the default domain (symbolic_opset9).
I can add a comment here to update to official onnx op when supported.